recipes/ttkwidgets/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "ttkwidgets" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 55e7eb0a86a3532fe38580b200a02c070d182f25a9f1c1aa1f22ff329c2e78f6
+
+build:
+  noarch: python 
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - pillow
+    - python >=3.6
+    - xorg-libx11 
```

Comment:
Maybe this should be a test dependency b/c it is not present in the docker image but usually we expect that X11 is present in most users machines.

Reply:
Actually, I did not think this dependency was needed, but since I could not build the commit without it, I had to add it. Should I just remove it?

Reply:
You were always right, this is only a test dependency, I got it wrong and thought it was a build dependency. I updated the recipe please have a look. Thank you @ocefpaf.